kernel/packages/unity-interface/UnityScene.ts

```diff
@@ -70,7 +70,11 @@ export class UnityScene<T> implements ParcelSceneAPI {
     const sceneId = getParcelSceneID(this)
     for (let i = 0; i < actions.length; i++) {
       const action = actions[i]
-      nativeMsgBridge.SendNativeMessage(sceneId, action)
+      try {
+        nativeMsgBridge.SendNativeMessage(sceneId, action)
+      } catch (e) {
+        this.logger.error(e)
```

Comment:
I think we should throw this error all the way to the scene and then shut it down. Keeping other scenes running

Reply:
That's a really good point but I need some of the platform guys to help me with this. If they cannot do it in this PR, is it okay to merge it  as it is and create an issue to be taken down whenever time allows?

Reply:
Why shut it down? Why not just ignore the message?

Reply:
As a dev I would prefer the whole scene to be shut down. You cannot really foresee the consequences of ignoring a single message in the scene flow and it could run the experience, cause buggy behaviour or even cost you money.

Reply:
I am concerned about the final-user experience thinking that decentraland explorer is broken when the fault is from the scene dev, I think that we should kill the scene and give some feedback to the user rather than just ignore the message and making some strange behaviours

Reply:
If we shut down the scene, we should display something in the scene itself (in 3D) telling that we had to shut it down or that it had fatal errors.

I preffer ignoring the problematic message and trigger an error in browser console telling exactly **which scene violated the kernel-client messages max-size** and telling that the scene **may display unwanted behaviours** (because of the skipped kernel-client message).

Reply:
I agree with Pravus:
- If we go with the "shut down" option, I think it is completely needed to design any visual feedback that indicates to the final user what is happening. If this feedback does not exist, the experience for the user would be weird and not beneficial for us.
- If we go with the "ignoring message" option, I like what Pravus suggest about simply throw an error message in the console warning about possible unexpected behaviours.

Reply:
Let's explore the problem space a bit:

Currently this can only happen in components where the payload contains a user-made string OR maybe with some kind of hack. Let's assume the first.

As far as I'm looking, the components with user made strings in their payloads are:
- TextShape (text)
- LoadableShapes (url)
- NFTShapes (url)
- UIShapes (string alignment, string positions, text)

(Note of color: All entity ids pass as ints now instead of strings due to the mapping wrapper in `SceneRuntime.ts`, so no way of hacking ids here I think.)

The components that would have most issues with a message missing due to interruptions in the scene interaction would be the UUIDComponents. But as I see so far, UUIDComponents has no user string fields, so no risk here. 

I think we can safely ignore the message, with the only risk being a visual component not being updated in that cycle. The state lies in the worker so no risk of state corruption either.

I see no good reason for shutting down the scene because is not the convention on how we handle issues so far on the world, this will come across as confusing for all the personas in decentraland. Also, its a bit funny that we aren't shutting down scenes for performance reasons but we do so for this microscopic edge case.

In any case we should not only terminate the worker but send a scene remove message, or the UX will suffer. For the feedback, we can use our already existing notification system (top middle popup message).